internal/binutils/binutils_test.go

```diff
@@ -220,10 +220,16 @@ func TestDisasm(t *testing.T) {
 	if runtime.GOOS != "linux" && runtime.GOOS != "darwin" {
 		t.Skip("This test only works on Linux or Mac")
 	}
-	testDisasm(t, true)
 	testDisasm(t, false)
 }
 
+func TestDisasmIntelSyntax(t *testing.T) {
+	if (runtime.GOOS != "linux" && runtime.GOOS != "darwin") || runtime.GOARCH != "amd64" {
+		t.Skip("This test only works on x86_64 Linux or x86_64 OSX as it tests Intel asm syntax")
```

Comment:
nit: reading up a bit, it looks like the right name for the OS on Mac is "macOS" - see https://en.wikipedia.org/wiki/MacOS#macOS. So OSX -> macOS here and Mac -> macOS above.

Reply:
Done.